zerver/openapi/zulip.yaml

```diff
@@ -5452,7 +5452,7 @@ paths:
         whether there are more messages matching the query that were not
         returned due to the `num_before` and `num_after` limits.
 
-        We recommend using `num_before <= 1000` and `num_after <= 1000` to
+        We recommend using `"num_before": 1000` and `"num_after": 1000` to
```

Comment:
This is a semantic change I don't think we intend: the recommendation is to set `num_before` and `num_after` to no more than 1000, not necessarily to 1000. Setting to 100, for example, may often be a better choice.

Could say
> We recommend setting `num_before` and `num_after` to ≤1000 to avoid […]

or with more words and fewer symbols:
> We recommend setting `num_before` and `num_after` to no more than 1000, to avoid […]

Reply:
I went with the second option with more words and fewer symbols.